src/map/packets/quest_mission_log.h

```diff
@@ -53,38 +98,60 @@
 #define ADO_COMPLETE		0xF8
 #define COA_COMPLETE		0x0108
 
-//Mission Log Types
 #define MISS_COMPLETE		0xD0
 #define MISS_CURRENT		0xFFFF
 
-// MISS_CURRENT	0xFF исключая Add-on Scenarios
+#define EXP_MISS_COMPLETE   0xD8
+#define EXP_MISS_CURRENT    0x80
 
-#define EXP_MISS_COMPLETE               0xD8
-#define EXP_MISS_CURRENT                0x80
+#define CMPGN_MISS_UN       0x30
+#define CMPGN_MISS_DEUX     0x38
 
-#define CAMPAIGN_MISSION_UN             0x30
-#define CAMPAIGN_MISSION_DEUX           0x38
+// Status Types
+#define STATUS_QUEST_CURR   0x01
+#define STATUS_QUEST_COMP   0x02
+#define STATUS_MISS_CURR    0x03
+#define STATUS_MISS_COMP    0x04
 
-// Mission Logs
-#define MISSION_SANDORIA                0x0B
-#define MISSION_BASTOK                  0x0C
-#define MISSION_WINDURST                0x0D
-#define MISSION_ZILART                  0x0E
-#define MISSION_TOAU                    0x0F
-#define MISSION_WOTG                    0x10
-#define MISSION_COP                     0x11
-
-#define MISSION_ASSAULT                 0x12
-
-#define MISSION_CAMPAIGN                0x13
-#define MISSION_CAMPAIGN2               0x14
-
-#define MISSION_ACP                     0x15
-#define MISSION_AMK		                0x16
-#define MISSION_ASA		                0x17
-
-#define MISSION_SOA					    0x18
-#define MISSION_ROV                     0x19
+// Log Types
+const int LOG_TYPES[34][7] =
```

Comment:
> I know how to avoid that

You know, I'm feeling a bit sheepish right now... because there probably is a way to not go straight to the declared Lunar method and I'm not familiar enough with Lunar to recognize it (my guess: renaming the current binding, and having new Lua-intermediary take its place, calling the new binding?). Going to see if I can figure it out before you tell me!